mlir-tensorrt/compiler/lib/Conversion/TensorRTToTensorRTRuntime/TensorRTToTensorRTRuntime.cpp

```diff
@@ -93,6 +93,8 @@ convertCallOp(Operation *op, IRRewriter &rewriter,
   SmallVector<int64_t> hostTensorArgs;
   for (auto [idx, arg] : llvm::enumerate(trtFunc.getArguments())) {
     const TensorKindLattice *kind = solver.lookupState<TensorKindLattice>(arg);
+    if (!isa<RankedTensorType>(arg.getType()))
```

Comment:
This is required since function argument could be a stream or context.

Reply:
A `func.func` that is the callee of `tensorrt.call|call_alloc` should only have tensor arguments. You have observed otherwise?

Reply:
Actually. Weirdly, we do not require this check anymore. This was from my initial PR, so it looks like this is not required anymore.